Cargo.toml

```diff
@@ -1,10 +1,18 @@
 [package]
 name = "u4pak"
 version = "1.3.0"
-authors = ["Mathias Panzenböck <grosser.meister.morti@gmx.net>"]
+authors = ["Mathias Panzenböck <grosser.meister.morti@gmx.net>", "L. Sprengel <l.sprengel@pm.me>"]
```

Comment:
Hope its ok, that I added myself here :)

Reply:
Of course!